repos/fdbt-site/src/pages/viewPassengerTypes.tsx

```diff
@@ -19,21 +20,23 @@ interface PassengerTypeProps {
     singlePassengerTypes: SinglePassengerType[];
     groupPassengerTypes: FullGroupPassengerType[];
     referer: string | null;
+    productsUsingPassengerType: { errors: ErrorInfo[]; productsUsingPassengerType: string[]; passengerName: string; } | null;
```

Comment:
we are only using the errors, not the other attributes like productsUsingPassengerType and passengerName, remove them

Reply:
sure working on it

Reply:
[c54320c](https://github.com/fares-data-build-tool/create-data/pull/578/commits/c54320c83560a526531b3cd112f6764e27f23dfe)